### ISSUE TITLE
Sympy sp.lambdify fails with sp.im but works with sp.re, TypeError: not enough arguments for format string

### ISSUE THREAD

**Issue by ManuelMBULL** [sympy/sympy#17625]
Python3 version is 3.7.3-1
Sympy version is 1.3-2
Scipy version is 1.2.2-7

Here is the orig.py code:

```
import sympy                     as     sp
from   sympy.utilities.iterables import flatten

a,b,c = sp.symbols( 'a,b,c', real = True )

v1 = sp.re( ( ( a + b*sp.I ) * sp.exp( c*sp.I ) )**3 )
v2 = sp.im( ( ( a + b*sp.I ) * sp.exp( c*sp.I ) )**3 )
print( 'v1 expression: ', v1 )
print( 'v2 expression: ', v2 )

lamb_v1 = sp.lambdify( flatten([a,b,c]), v1, modules='scipy' )
print('Trying lamb_v1: ', lamb_v1( *flatten([1,2,3]) ) )
lamb_v2 = sp.lambdify( flatten([a,b,c]), v2, modules='scipy' )
print('Trying lamb_v2: ', lamb_v2( *flatten([1,2,3]) ) )
```

And the report is:

```
v1 expression:  re((a + I*b)**3*exp(3*I*c))
v2 expression:  im((a + I*b)**3*exp(3*I*c))
Trying lamb_v1:  10.84666985121496
Traceback (most recent call last):
  File "orig.py", line 13, in <module>
    lamb_v2 = sp.lambdify( flatten([a,b,c]), v2, modules='scipy' )
  File "/usr/lib/python3/dist-packages/sympy/utilities/lambdify.py", line 470, in lambdify
    funcstr = funcprinter.doprint(funcname, args, expr)
  File "/usr/lib/python3/dist-packages/sympy/utilities/lambdify.py", line 695, in doprint
    funcbody.append('return ({})'.format(self._exprrepr(expr)))
  File "/usr/lib/python3/dist-packages/sympy/printing/codeprinter.py", line 101, in doprint
    lines = self._print(expr).splitlines()
  File "/usr/lib/python3/dist-packages/sympy/printing/printer.py", line 287, in _print
    return getattr(self, printmethod)(expr, **kwargs)
  File "/usr/lib/python3/dist-packages/sympy/printing/pycode.py", line 489, in _print_im
    return "%s(%s)" % (self._module_format('numpy.imag', self._print(expr.args[0])))
TypeError: not enough arguments for format string
```

**Comment by asmeurer** [comment]
I think this has already been fixed in master.

**Comment by asmeurer** [comment]
It looks like it's fixed in sympy 1.4 as well. If you upgrade the problem should go away.
